Add a general article on osu! forums. Add forum-specific stubs (#6454)
ppy/osu-wiki: wiki/Community/Forum/Modding_Queues/en.md

> ---

Reviewer: if you feel it's necessary, mention& link the recently added subforums

stub: true
---

# Modding Queues

<!-- TODO: this needs to be linked from other related articles, for example, "how to get mods" -->

[**Modding Queues**](https://osu.ppy.sh/community/forums/60) is a forum dedicated to [beatmap review](/wiki/Modding) requests, or mods. It consists of forum threads, or *queues* belonging to different modders and mappers. Every queue has individual requirements and limits on obtaining the mods, which must be respected by its visitors in order to receive a mod.

## Queue rounds

> Before the start of every round, the queue's owner posts a short announcement, which typically contains:

Reviewer: this needs a sentence before it explaining what a "round" is and why most queues operate that way

---
stub: true
---

# Modding Queues

<!-- TODO: this needs to be linked from other related articles, for example, "how to get mods" -->

[**Modding Queues**](https://osu.ppy.sh/community/forums/60) is a forum dedicated to [beatmap review](/wiki/Modding) requests, or mods. It consists of forum threads, or *queues* belonging to different modders and mappers. Every queue has individual requirements and limits on obtaining the mods, which must be respected by its visitors in order to receive a mod.

> Since 2011-11-30,<!-- TODO: add https://github.com/ppy/osu-web/issues/8256 as a footnote when they're supported --> each game mode has its own subforum for queues, with osu! taking up main section of the forum:

Reviewer 2: that's table format (also wrong decade rofl)

```suggestion
Since November 30, 2021,<!-- TODO: add https://github.com/ppy/osu-web/issues/8256 as a footnote when they're supported --> each game mode has its own subforum for queues, with osu! taking up the main section of the forum:
```

but isn't this more fit for the history articles anyway?

```suggestion
Each game mode has its own subforum for queues, with osu! taking up the main section of the forum:
```

Author: maybe, I felt like it was important at the moment of editing. feel free to take it out

- [osu!](https://osu.ppy.sh/community/forums/60)
- [osu!taiko](https://osu.ppy.sh/community/forums/121)
- [osu!catch](https://osu.ppy.sh/community/forums/122)
- [osu!mania](https://osu.ppy.sh/community/forums/123)

## Queue rounds

> To keep their workload at appropriate level, most modders only accept requests during short periods called rounds. Before the start of every round, the queue's owner posts a short announcement, which typically contains:

Reviewer 2: ```suggestion
To keep their workload at an appropriate level, most modders only accept requests during short periods called rounds. Before the start of every round, the queue's owner posts a short announcement, which typically contains:
```

- The number of maps that may be accepted in this round

> - Requirements specific to this round

Reviewer 2: ```suggestion
- Requirements specific to the current round
```
